Resources/Prototypes/_RMC14/Actions/Xeno/xeno_self_actions.yml

```
@@ -307,8 +307,8 @@
 - type: entity
   id: ActionXenoThrowParasite
   parent: ActionXenoBase
-  name: Throw Parasite
-  description: Retrieves a parasite from your storage if you have an empty hand. Using this on a parasite on the ground will pickup all nearby parasites. Targeting a tile with a parasite in hand will throw it up to 4 tiles away, where the parasite will be stunned for 5 seconds. Using a parasite in hand will add it to your storage. [color=green]Parasites won't die due to time while you hold or carry them.[/color]
+  name: Throw Child
```

Two thirds of this ability have nothing to do with throwing, we need a better name for this. No wonder people got confused how carrier works. This name is just a suggestion, idk
```suggestion
  name: Carrier's Bond
```

---

done